src/main/java/uk/gov/crowncommercial/dts/scale/cat/utils/TendersAPIModelUtils.java

```diff
@@ -59,6 +60,17 @@ public EventSummary buildEventSummary(final String eventId, final String name,
 
     return eventSummary;
   }
+  public EventSummary buildEventSummary(final String eventId, final String name,
```

Comment:
Sonar is complaining that there are 9 params in this method (>7). Think it's also generally good practice to avoid overloaded methods if possible.

I'd maybe suggest just calling the existing method and setting the tenderPeriod in the ProcurementProjectService.. ?

Reply:
done